graphql-dgs-spring-boot-oss-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/autoconfig/DgsAutoConfiguration.kt

```diff
@@ -29,7 +29,6 @@ import com.netflix.graphql.dgs.context.GraphQLContextContributor
 import com.netflix.graphql.dgs.context.GraphQLContextContributorInstrumentation
 import com.netflix.graphql.dgs.exceptions.DefaultDataFetcherExceptionHandler
 import com.netflix.graphql.dgs.internal.*
-import com.netflix.graphql.dgs.internal.DefaultDgsQueryExecutor.ReloadSchemaIndicator
```

Comment:
file comment: is this supposed to be removed in favor of `DgsSpringGraphQLAutoConfiguration`?
module comment: I guess the whole `graphql-dgs-spring-boot-oss-autoconfigure` should be removed, right?

Reply:
yes, thanks, I removed it from settings.gradle but forgot to remove the files.